### DISCUSSION TITLE
Autohotkey + Python script to shorten urls from clipboard.

### DISCUSSION THREAD

**Discussion by Today20092** [shlinkio/discussions#141] [Show and tell]
I made this autohotkey script that calls a python script with the contents of the keyboard. It is not the greatest coding, but it works for me, maybe someone can use it or make it better.

https://github.com/Today20092/Shlink-API

**Comment by acelaya** [comment]
That URL returns a 404 to me. Maybe it's not a public repository?

**Reply by Today20092** [reply]
Fixed 🔥
